Compile error on rust nightly 2022-04-06

```
error[E0034]: multiple applicable items in scope
   --> /tmp/.cargo/registry/src/github.com-1ecc6299db9ec823/lightning-0.0.102/src/ln/msgs.rs:962:32
    |
962 |             DecodeError::Io(ref e) => e.fmt(f),
    |                                         ^^^ multiple fmt found
    |
    = note: candidate #1 is defined in an impl of the trait std::fmt::Display for the type std::io::ErrorKind
    = note: candidate #2 is defined in an impl of the trait Debug for the type std::io::ErrorKind
```

--- a/ln-gateway/Cargo.toml
+++ b/ln-gateway/Cargo.toml
@@ -11,7 +11,7 @@ async-trait = "0.1.52"
 clightningrpc = { git = "https://github.com/elsirion/rust-clightning-rpc", rev = "28b0b062dfb7d758a3a0753c274febe4a7faa584" }
 futures = "0.3.21"
 hex = "0.4.3"
-lightning-invoice = "0.11.0"
+lightning-invoice = "0.14.0"
 minimint = { path = "../minimint" }
 minimint-api = { path = "../minimint-api" }
 mint-client = { path = "../mint-client" }

--- a/mint-client/Cargo.toml
+++ b/mint-client/Cargo.toml
@@ -14,8 +14,8 @@ bitcoin = "0.27.0"
 bitcoin_hashes = "0.10.0"
 futures = "0.3.9"
 hex = "0.4.3"
-lightning-invoice = "0.11.0"
-lightning = "0.0.103"
+lightning-invoice = "0.14.0"
+lightning = "0.0.106"
 miniscript = "6.0.0"
 minimint = { path = "../minimint" }
 minimint-api = { path = "../minimint-api" }

--- a/modules/minimint-ln/Cargo.toml
+++ b/modules/minimint-ln/Cargo.toml
@@ -11,8 +11,8 @@ bincode = "1"
 bitcoin_hashes = "0.10"
 futures = "0.3.21"
 itertools = "0.10.1"
-lightning = "0.0.103"
-lightning-invoice = "0.10"
+lightning = "0.0.106"
+lightning-invoice = "0.14.0"
 minimint-api = { path = "../../minimint-api" }
 secp256k1 = "0.20"
 serde = {version = "1.0.130", features = [ "derive" ] }

